src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -63,11 +63,6 @@ def handle_exception(ex):  # pylint: disable=too-many-locals, too-many-statement
     from requests.exceptions import SSLError, HTTPError
     from azure.cli.core import azclierror
     from msal_extensions.persistence import PersistenceError
-    import traceback
-
-    logger.debug("azure.cli.core.util.handle_exception is called with an exception:")
-    # Print the traceback and exception message
-    logger.debug(traceback.format_exc())
```

Comment:
I'd like to confirm that when the error type thrown is `CLIError`, even if the logic of printing traceback here is removed, the traceback will still be printed in `azclierror.UnclassifiedUserFault(error_msg)`, right?

https://github.com/Azure/azure-cli/blob/325dfa64d8b77bfddb3b087162ad98acee5037ab/src/azure-cli-core/azure/cli/core/util.py#L109-L111

Reply:
Yes, as long as `azure.cli.core.azclierror.AzCLIError.print_error` is called, the traceback will be printed by

https://github.com/Azure/azure-cli/blob/a08283a73045bc06ee8d35dc99e2a215690706a5/src/azure-cli-core/azure/cli/core/azclierror.py#L66-L68